developers/wcs/embeddings/index.md

```diff
@@ -1,293 +1,53 @@
-    'tertiaryColor': '#edf2f7',
-    'fontFamily': 'Inter, system-ui, sans-serif',
-    'fontSize': '14px',
-    'lineHeight': '1.4',
-    'nodeBorder': '1px',
-    'mainBkg': '#ffffff',
-    'clusterBkg': '#f8fafc'
-  }
-}}%%
-
-flowchart LR
-    %% Style definitions
-    classDef systemBox fill:#f8fafc,stroke:#3182ce,stroke-width:1.5px,color:#2d3748,font-weight:bold
-    classDef weaviateBox fill:#f8fafc,stroke:gray,stroke-width:0.5px,color:#2d3748,font-weight:bold
-    classDef cloudBox fill:white,stroke:#48bb78,stroke-width:2px,color:#553c9a,font-weight:bold
-    classDef providerBox fill:#f8fafc,stroke:gray,stroke-width:0.5px,color:#2d3748,font-weight:bold
-    classDef component fill:white,stroke:#a0aec0,stroke-width:1px,color:#2d3748
-    classDef edgeLabel fill:white,stroke:#e2e8f0,stroke-width:1px,color:#4a5568
-
-    %% Weaviate Cloud container
-    subgraph cloud["☁️ Weaviate Cloud"]
-        %% Weaviate Embeddings section
-        subgraph provider["Weaviate\nEmbeddings"]
-            inference["🤖 Inference\nAPI"]
-        end
-
-        %% Weaviate section
-        subgraph weaviate["Weaviate Cloud instance"]
-            vectorizer["🔌 Weaviate Embeddings\nIntegration"]
-            core["💾 Data &\nvector store"]
-        end
-    end
-
-    %% User System
-    subgraph user["🖥️ User System"]
-        data["📄 Data"]
-    end
-
-    %% Connections with curved edges
-    data --->|"1. Insert\nobjects"| core
-    core --->|"2. Request\nvector"| vectorizer
-    vectorizer --->|"3. Request\nvector"| inference
-    inference --->|"4. Vector"| vectorizer
-    vectorizer --->|"5. Vector"| core
-
-    %% Apply styles
-    class user systemBox
-    class weaviate weaviateBox
-    class cloud cloudBox
-    class provider providerBox
-    class data,core,vectorizer,inference component
-
-    %% Linkstyle for curved edges
-    linkStyle default stroke:#718096,stroke-width:3px,fill:none,background-color:white
-```
+![Weaviate Embeddings flowchart](../img/weaviate-embeddings-flowchart.png "Weaviate Embeddings flowchart")
 
 With Weaviate Embeddings, you can generate embeddings for your data and queries directly from a Weaviate Cloud database instance.
 
-This means you can perform semantic, vector and hybrid searches without the need to externally generate vector embeddings, or manage additional model providers.
+This means you can perform [keyword](/developers/weaviate/search/bm25), [vector](/developers/weaviate/search/similarity) and [hybrid searches](/developers/weaviate/search/hybrid) without the need to externally generate vector embeddings, or manage additional model providers.
 
-Weaviate Embeddings is fully integrated with Weaviate Cloud, so you can manage your data and embeddings in one place.
+Weaviate Embeddings is fully integrated with Weaviate Cloud, so you can manage your data and embeddings in one place. Weaviate Embeddings is available for Weaviate Cloud instances and is currently not available when running a local instance.
```

Comment:
I think this page paragraph and page says "Weaviate Embeddings" a lot 😅, and reads more like marketing copy than technical documentation. Let's reduce these references where possible.

Reply:
In fact, I'm not sure that we need this paragraph. Could be an INFO admonition with `Weaviate Embeddings is only available for use with Weaviate Cloud instances.`?

Reply:
Replaced it with an admonition which is even better because it puts more attention on the fact that Weaviate Embeddings are only available in WCD.